lib/templ.ml

```diff
@@ -184,12 +237,17 @@ and eval_simple_variable conf =
     )
   | "lang" -> conf.lang
   | "left" -> conf.left
+  | "link_next" ->
+      begin match Util.p_getenv conf.env "link_next" with
+      | Some vv -> vv
+      | None -> ""
+      end
   | "nl" -> "\n"
   | "nn" -> ""
+  | "plugins" -> (List.fold_left (fun s p -> (Filename.basename p) ^ "," ^ s) "" conf.plugins)
```

Comment:
I think there is an unnecessary ',' at the end of the string.
I would do it like this:
```
let l = List.map Filename.basename conf.plugins in
String.concat "," l